### DISCUSSION TITLE
Codeception support policy

### DISCUSSION THREAD

**Discussion by Jamesking56** [Codeception/Codeception#6616] [Q&A]
Hi,

What versions of Codeception are supported / unsupported? Is there a written support policy for Codeception?

Cheers.

**Comment by Naktibalda** [comment]
It depends on what you mean by "support" :)

There is no such policy, not much support either.

5.0 is current active version.
4.2 may receive some bugfixes, for example if it is has some critical bug, or someone raises pull request to fix a not so critical bug, or @DavertMik has a paying customer asking for some change, like https://github.com/Codeception/Codeception/pull/6569 , though it looks abandoned.

**Reply by Jamesking56** [reply]
I have a library for Codeception and I was wondering what versions of Codeception and of PHP it should support to try and match up with Codeception's own support policy

**Reply by Naktibalda** [reply]
It is completely up to you.

Not supporting Codeception 5 is very bad :)

**Reply by DavertMik** [reply]
It's always the best option to keep on latest changes with Codeception
